fujiji-server/src/repositories/user.js

```diff
@@ -0,0 +1,25 @@
+const { Sequelize } = require('sequelize');
+const { dbConfig } = require('../config/config');
+
+const sequelize = new Sequelize(dbConfig.NAME, dbConfig.USERNAME, dbConfig.PASSWORD, {
+  host: dbConfig.HOST,
+  dialect: 'mssql',
+  pool: {
+    max: 5,
+    min: 0,
+    idle: 10000,
+  },
+});
```

Comment:
We probably can extract this out and put it in separate module like `db.js` maybe (?)

and do something like this: `module.exports = sequelize` so we can re-use this by using `require`

Then, at the other file that we want to use we can just use this:
```
const sequelize = require('/path/to/module')
```

Reply:
I had tried that, but still didn't work correctly

Reply:
np at all! lol I was so confused to with module.exports the first time I use node

everything lgtm now!